lib/bbbevents/events.rb

```diff
@@ -152,7 +152,16 @@ def user_responded_to_poll_record_event(e)
       return unless attendee = @attendees[@externalUserId[intUserId]]
 
       if poll = @polls[poll_id]
-        poll.votes[@externalUserId[intUserId]] = poll.options[e["answerId"].to_i]
+        if poll.type == 'R-'
+          poll.votes[@externalUserId[intUserId]] = e["answer"]
+
+          # We want to merge the same responses
+          poll_set = poll.options.to_set
```

Comment:
Instead of using a set here, are we able to add the entries directly to the array? The BigBlueButton server should have already de-duplicated them - can we use the "answerId" as an index into the options array?